pyro/contrib/oed/eig.py

```diff
@@ -55,7 +60,7 @@ def posterior_entropy(y_dist, design):
         conditioned_model = pyro.condition(model, data=y_dict)
         SVI(conditioned_model, **vi_parameters).run(design)
         # Recover the entropy
-        return mean_field_guide_entropy(vi_parameters["guide"], design)  # TODO specify inference labels pattern here
+        return mean_field_guide_entropy(vi_parameters["guide"], [design], whitelist=target_labels)
```

Comment:
You can pick up entropy from only some sites now. Require mean field (as before)